app/models/textbook_transaction.rb

```diff
@@ -0,0 +1,14 @@
+class TextbookTransaction < ActiveRecord::Base
+	default_scope { order('created_at DESC') }
+
+	belongs_to :seller, :class_name => 'User', :foreign_key => 'seller_id'
```

Comment:
not sure if you need :foreign_key it should know it based on belongs_to :seller - try it to see if works